client/src/App.tsx

```diff
@@ -1,35 +1,15 @@
-import { useState } from 'react'
```

Comment:
You don't want this in App.tsx
make a Menu.tsx and put your code there.
then put Menu.tsx inside App.tsx to test it out.

Reply:
main reason is cuz we are gonna have other routes 
say /game or /room
So App.tsx is gonna need to have the components for each of those routes.
if that didn't make much sense lemme know and I can show you.

Reply:
MainMenu.tsx*